WebViewControl/WebView.CefResourceRequestHandler.cs

```diff
@@ -73,70 +69,18 @@ protected class CefResourceRequestHandler : ResourceRequestHandler, IDisposable
-
-                var extension = Path.GetExtension(resourcePath.Last()).ToLower();
+                var resource = ResourcesManager.LoadEmbeddedResource(url);
 
-                var resourceStream = TryGetResourceWithFullPath(resourceAssembly, resourcePath);
-                if (resourceStream != null) {
-                    resourceHandler.RespondWith(resourceStream, extension);
+                if (resource.ResourceStream != null) {
+                    resourceHandler.RespondWith(resource.ResourceStream, resource.Extension);
                 }
             }
 
-            protected Assembly ResolveResourceAssembly(Uri resourceUrl) {
-                if (assemblies == null) {
-                    assemblies = new Dictionary<string, Assembly>();
-                    AppDomain.CurrentDomain.AssemblyLoad += OnAssemblyLoaded;
-                }
-
-                var assemblyName = ResourceUrl.GetEmbeddedResourceAssemblyName(resourceUrl);
-                var assembly = GetAssemblyByName(assemblyName);
-
-                if (assembly == null) {
-                    if (newAssembliesLoaded) {
-                        // add loaded assemblies to cache
-                        newAssembliesLoaded = false;
-                        foreach (var domainAssembly in AppDomain.CurrentDomain.GetAssemblies()) {
-                            // replace if duplicated (can happen)
-                            assemblies[domainAssembly.GetName().Name] = domainAssembly;
-                        }
-                    }
-
-                    assembly = GetAssemblyByName(assemblyName);
-                    if (assembly == null) {
-                        // try load assembly from its name
-                        assembly = AppDomain.CurrentDomain.Load(new AssemblyName(assemblyName));
-                        if (assembly != null) {
-                            assemblies[assembly.GetName().Name] = assembly;
-                        }
-                    }
-                }
-
-                if (assembly != null) {
-                    return assembly;
-                }
-
-                throw new InvalidOperationException("Could not find assembly for: " + resourceUrl);
-            }
-
-            private Assembly GetAssemblyByName(string assemblyName) {
-                Assembly assembly;
-                assemblies.TryGetValue(assemblyName, out assembly);
-                return assembly;
-            }
-
             protected Stream TryGetResourceWithFullPath(Assembly assembly, IEnumerable<string> resourcePath) {
                 return ResourcesManager.TryGetResourceWithFullPath(assembly, resourcePath);
             }
 
-            private void OnAssemblyLoaded(object sender, AssemblyLoadEventArgs args) {
-                newAssembliesLoaded = true;
-            }
-
-            public void Dispose() {
-                AppDomain.CurrentDomain.AssemblyLoad -= OnAssemblyLoaded;
-            }
+            public void Dispose() { }
```

Comment:
Remove

Reply:
are we ever going to unsubscribe the event?

Reply:
WebView.cs is expecting the RequestHandler to be IDisposable

Reply:
Does not need anymore

Reply:
Doesn't neeed to